### PR TITLE
Creates a helper function to get selected chart

### PR DESCRIPTION
**Pull request by belcherj**
Adds a getSelectedChart helper function so that it can be used by all reports.

### Detailed test instructions:

 - Open orders chart and click through summary numbers

### CODE REVIEW

File: client/lib/get-selected-chart/index.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+/**
+ * Takes a chart name returns the configuration for that chart from and array of charts
+ *
+ * @param {string} chartName - the name of the chart to get configuration for
+ * @param {array} charts - list of charts for a particular report
+ * @returns {object} - chart configuration object
+ */
+export default function getSelectedChart( chartName, charts ) {
+	const chart = find( charts, { key: chartName } );
+	console.log( chartName, charts, chart );
```
Comment (coderkevin) [inline client/lib/get-selected-chart/index.js]: You'll need to remove this before merging.

File: client/lib/get-selected-chart/index.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+ * External dependencies
+ */
+import { find } from 'lodash';
+
+/**
+ * Takes a chart name returns the configuration for that chart from and array of charts
+ *
+ * @param {string} chartName - the name of the chart to get configuration for
+ * @param {array} charts - list of charts for a particular report
+ * @returns {object} - chart configuration object
```
Comment (coderkevin) [inline client/lib/get-selected-chart/index.js]: May want to mention that if the chart name isn't found that it returns the first in the array.

File: client/lib/get-selected-chart/index.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+ */
+export default function getSelectedChart( chartName, charts ) {
+	const chart = find( charts, { key: chartName } );
+	console.log( chartName, charts, chart );
+	if ( chart ) {
+		return chart;
+	}
+	return charts[ 0 ];
```
Comment (coderkevin) [inline client/lib/get-selected-chart/index.js]: Can we be sure that charts will never be empty?

File: client/lib/get-selected-chart/index.js (hunk 2 above)
Comment (belcherj) [reply client/lib/get-selected-chart/index.js]: Added to the function description

File: client/lib/get-selected-chart/index.js (hunk 1 above)
Comment (belcherj) [reply client/lib/get-selected-chart/index.js]: Removed

**Pull request merged**